simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/exception/AvailabilityStateVariableAccessor.java

```diff
@@ -49,12 +52,21 @@ public String getVariableName() {
     }
 
     private GetVariableResult getActualValue(Component component, Variable variable, GetVariableDatum.AttributeType attributeType) {
-        return new GetVariableResult()
+        Integer evseId = component.getEvse().getId();
+        Optional<Evse> optionalEvse = getStation().getState().tryFindEvse(evseId);
```

Comment:
It would be better to have a method that returns boolean such as: `hasEvse(evseId)`